openshift/e2e-tests-openshift.sh

```diff
@@ -20,6 +21,19 @@ readonly OPENSHIFT_BUILD_NAMESPACE=${OPENSHIFT_BUILD_NAMESPACE:-tektoncd-build-$
 # test-git-volume: `"gitRepo": gitRepo volumes are not allowed to be used]'
 declare -ar SKIP_YAML_TEST=(test-git-volume)
 
+function replaceImagesWithCIBuilds() {
+  RELEASE_YAML=$1
+  REGISTRY_PREFIX=$2
+
+  # Substitute images using the images built by OpenShift CI
+  # Ref: https://github.com/openshift/release/pull/21940
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-triggers-controller|${TEKTONCD_TRIGGERS_CONTROLLER}|g" ${RELEASE_YAML}
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-triggers-eventlistenersink|${TEKTONCD_TRIGGERS_EVENTLISTENERSINK}|g" ${RELEASE_YAML}
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-pipeline-interceptors|${TEKTONCD_TRIGGERS_INTERCEPTORS}|g" ${RELEASE_YAML}
```

Comment:
tektoncd-pipeline-interceptors :thinking:

Reply:
Good catch. Fixed this.